### PR TITLE
C1X adapter updated to Prebid v1.0

### PR DESCRIPTION
**Pull request by CathyHuangtw**
## Type of change
- [ ] Bugfix
- [ ] Feature
- [x] New bidder adapter  
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other

## Description of change
Update C1X adapter to Prebid v1.0

- test parameters for validating bids
```
{
   bidder: 'c1x',
   params: {
      siteId: '9999',
      pixelId: '12345',
      floorPriceMap: {
        '300x250': 0.20,
        '300x600': 0.30
      }, //optional
   }
}
```
- cathy@c1exchange.com
- [x] official adapter submission

### CODE REVIEW

File: modules/c1xBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,168 @@
+// import CONSTANTS from 'src/constants.json';
```
Comment (jaiminpanchal27) [inline modules/c1xBidAdapter.js]: Remove unused import

Comment (CathyHuangtw) [reply modules/c1xBidAdapter.js]: @jaiminpanchal27  Done. Thank you!

File: modules/c1xBidAdapter.js [hunk 2]
```diff
@@ -0,0 +1,167 @@
+import { registerBidder } from 'src/adapters/bidderFactory';
+import * as utils from 'src/utils';
+import { userSync } from 'src/userSync';
+
+const BIDDER_CODE = 'c1x';
+const URL = 'http://18.217.214.190:8080/ht';
```
Comment (mkendall07) [inline modules/c1xBidAdapter.js]: this needs to support https.

**Comment by jaiminpanchal27** [comment]
@CathyHuangtw 
We also need docs PR to merge this. Please submit a PR to the [docs repo](https://github.com/prebid/prebid.github.io) to add a file for your adapter to the [bidders directory](https://github.com/prebid/prebid.github.io/tree/master/dev-docs/bidders) so your adapter's params will appear on the [bidders page](http://prebid.org/dev-docs/bidders.html). Thank you for contributing

**Comment by jaiminpanchal27** [comment]
@CathyHuangtw Any updates ?

**Comment by CathyHuangtw** [comment]
@jaiminpanchal27 thank you for the reminder. I just updated our endpoint URL to our production endpoint which supports HTTPS. I am going to submit a PR to docs repo now.

**Comment by CathyHuangtw** [comment]
https://github.com/prebid/prebid.github.io/pull/578
I've submitted a PR to the docs repo.
Please let me know if you have any questions. @jaiminpanchal27

**Comment by jaiminpanchal27** [comment]
@mkendall07's  requested changes are done. Hence merging.

**Pull request merged**